src/components/Form/Checkbox.tsx

```diff
@@ -127,6 +127,15 @@ export class Checkbox extends Component<CheckboxProps> {
       );
     }
 
+    if (
+      ('name' in this.props && name === '') ||
+      ('value' in this.props && value === '')
+    ) {
+      logger.warn(
+        'Name or value props should not have empty values. Provide valid values.',
```

Comment:
Maybe rephrase like "Name and value props should have non-empty values if provided."

Reply:
Good suggestion. Now fixed.